test/perf/concatMap.js

```diff
@@ -44,11 +45,14 @@ suite
 	.add('most', function(deferred) {
 		runners.runMost(deferred, most.from(a).concatMap(most.from).reduce(sum, 0));
 	}, options)
-	.add('rx', function(deferred) {
+	.add('rx 4', function(deferred) {
 		runners.runRx(deferred, rx.Observable.fromArray(a).concatMap(rx.Observable.fromArray).reduce(sum, 0));
 	}, options)
+  .add('rx 5', function(deffered) {
+    runners.runRx5(deffered, rxjs.Observable.fromArray(a).concatMap(function(x) {return rxjs.Observable.fromArray(x)}).reduce(sum, 0))
+  }, options)
 	.add('kefir', function(deferred) {
-		runners.runKefir(deferred, kefirFromArray(a).flatMapConcat(kefirFromArray).reduce(sum, 0));
+		runners.runKefir(deferred, kefirFromArray(a).flatMapConcat(kefirFromArray).scan(sum, 0));
```

Comment:
Seems like this should also call `.last()` to be truly [equivalent to `reduce`](https://github.com/rpominov/kefir/blob/3371d4b0dc44ba35610c6f06a833a139be03a11a/changelog.md#210-28042015)